lib/ansible/modules/windows/win_domain_group.ps1

```diff
@@ -0,0 +1,354 @@
+#!powershell
+# This file is part of Ansible
+#
+# (c) 2017, Jordan Borean <jborean93@gmail.com>, and others
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$ErrorActionPreference = "Stop"
+
+$params = Parse-Args -arguments $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $Params -name "_ansible_diff" -type "bool" -default $false
+
+$name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true
+$display_name = Get-AnsibleParam -obj $params -name "display_name" -type "str"
+$username = Get-AnsibleParam -obj $params -name "username" -type "str"
+$password = Get-AnsibleParam -obj $params -name "password" -type "str" -failifempty ($username -ne $null)
+$description = Get-AnsibleParam -obj $params -name "description" -type "str"
+$category = Get-AnsibleParam -obj $params -name "category" -type "str" -validateset "distribution","security"
+$scope = Get-AnsibleParam -obj $params -name "scope" -type "str" -validateset "domainlocal","global","universal"
+$managed_by = Get-AnsibleParam -obj $params -name "managed_by" -type "str"
+$attributes = Get-AnsibleParam -obj $params -name "attributes"
+$path = Get-AnsibleParam -obj $params -name "path" -type "str"
```

Comment:
I always thought `Path` was a strange choice for the PS stuff (I suspect they're trying to make things feel the same across PS cmdlets?)- I like dag's suggestion, since aliases are cheap... How about `organizational_unit` with `ou` and `path` as aliases (and standardize on that for docs/future AD modules)? Others may disagree, but I tend to favor documenting with more descriptive names, as it makes both the playbook and docs a little easier to read (both should be *read* much more often than *written*). Happy to discuss further though if anyone disagrees strongly.